### ISSUE TITLE
All switchers will stuck if a single key is used to trigger task switch.

### ISSUE THREAD

**Issue by ndgnuh** [WayfireWM/wayfire#1271]
**Describe the bug**
All switchers will stuck if a single key is used to trigger task switch.

**To Reproduce**
1. Enable plugin switcher or fast switcher
2. Set the switch key to a single key (like `<F3>`)
3. Use the switcher, the switcher will stuck and Wayfire have to be killed

**Expected behavior**
Not stuck

**Screenshots or stacktrace**
I don't know how to do screenshots because the whole desktop was stuck.

Here's a piece of log: [wflog.txt](https://github.com/WayfireWM/wayfire/files/7060293/wflog.txt)

**Wayfire version**
```
wayfire --version
0.7.2
```

Thanks!

**Comment by ammen99** [comment]
Yeah, switchers rely on a modifier to function properly. Otherwise, we don't know when to stop the animations. We could add some special-casing in switcher and fast-switcher so that they immediately close after switching if there is no modifier in the binding.
